tools/java/org/unicode/cldr/api/XmlDataSource.java

```diff
@@ -1,24 +1,8 @@
 package org.unicode.cldr.api;
 
-import com.google.common.base.CharMatcher;
-import com.google.common.base.Splitter;
-import com.google.common.base.Strings;
-import com.google.common.collect.HashMultiset;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.ImmutableSortedMap;
-import com.google.common.collect.Maps;
-import com.google.common.collect.Multiset;
-import org.unicode.cldr.util.CachingEntityResolver;
-import org.xml.sax.Attributes;
-import org.xml.sax.ErrorHandler;
-import org.xml.sax.InputSource;
-import org.xml.sax.SAXException;
-import org.xml.sax.SAXParseException;
-import org.xml.sax.XMLReader;
-import org.xml.sax.helpers.DefaultHandler;
-import org.xml.sax.helpers.XMLReaderFactory;
+import static com.google.common.base.CharMatcher.whitespace;
```

Comment:
We should definitely figure out the expected import order (I imported my IntelliJ setup from Eclipse so I thought it was being consistent, but if you have a significantly different setup, we'll have major churn every time someone makes a change.

We should also define a set of formatting options that work well on both Eclipse and IntelliJ and document them somewhere.

As a data point, Google recently went with a purely "simpler is better" approach and just list two blocks of imports (static, then class imports) with everything in each group just being alphabetical. Having "preferred" groups just gets messy, and mostly these things are handled by IDEs now anyway.

Reply:
> We should also define a set of formatting options that work well on both Eclipse and IntelliJ and document them somewhere.

@yumaoka has done our eclipse formatting before. 

Yes, I'll reapply this with minimal changes.

Reply:
.. using emacs as the IDE

Reply:
Fair enough. Just going with the simplest ordering we can (everything in a block, ordered lexicographically) will almost certainly be something that all editors/IDEs will support.

Reply:
@hagbard Could you add a comment on https://unicode-org.atlassian.net/browse/CLDR-9327 or else open  a new ticket?